instrumentation/netty/netty-4-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4/common/AbstractNettyChannelPipelineInstrumentation.java

```diff
@@ -176,16 +179,18 @@ public static void removeHandler(
           handler = pipeline.removeLast();
         }
       }
+      return handler;
     }
   }
 
   @SuppressWarnings("unused")
   public static class AddAfterAdvice {
 
     @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static void addAfterHandler(
-        @Advice.This ChannelPipeline pipeline,
-        @Advice.Argument(value = 1, readOnly = false) String name) {
+    @Advice.AssignReturned.ToArguments(@ToArgument(1))
+    public static String addAfterHandler(
+        @Advice.This ChannelPipeline pipeline, @Advice.Argument(value = 1) String nameArg) {
+      String name = nameArg;
```

Comment:
[for reviewer] here using an intermediate variable is required to keep the advice work with "inlined" and "indy", that's probably a minor side-effect of using `@Advice.AssignReturned.ToArguments` with an inlined advice.